class-wc-calypso-bridge.php

```diff
@@ -123,6 +123,8 @@ public function includes() {
 		require_once WC_CALYPSO_BRIDGE_PLUGIN_PATH . '/includes/class-wc-calypso-bridge-free-trial-payment-task.php';
 		require_once WC_CALYPSO_BRIDGE_PLUGIN_PATH . '/includes/free-trial/class-wc-calypso-bridge-free-trial-orders-changes.php';
 		require_once WC_CALYPSO_BRIDGE_PLUGIN_PATH . '/includes/free-trial/class-wc-calypso-bridge-free-trial-wc-payments.php';
+		require_once WC_CALYPSO_BRIDGE_PLUGIN_PATH . '/includes/class-wc-calypso-bridge-frontend-free-trial.php';
```

Comment:
Nit: Do you think this file could have a more representative naming? I think we could either have a generic controller that does multiple simple modifications for free trial, or specific modifications just for the customer screen.

Reply:
@ilyasfoo Good suggestion. Um..I can't think of a good naming since we don't have many use cases yet. I'm considering moving this class into `free-trial` and treating it like any other free-trial classes. What do you think?

Reply:
@moon0326 Sounds good!

Reply:
@ilyasfoo Updated in https://github.com/Automattic/wc-calypso-bridge/pull/917/commits/26ac35a7fd469ef905cd3dc291600a2989771403

Reply:
@moon0326  Excellent, lgtm!